ampligraph/datasets/abstract_dataset_adapter.py

```diff
@@ -98,30 +98,17 @@ def get_next_batch(self, batches_count=-1, dataset_type="train"):
             indicates which dataset to use
         batches_count: int
             number of batches per epoch (default: -1, i.e. uses batch_size of 1)
```

Comment:
The batches_count=-1 -> batch_size=1 is a little awkward but I don't see a better way of implementing it other than switching entirely to batch_size.

Reply:
(Which I'm not suggesting we do.)